src/domain_features/components/Transactions/transactionsTableHeaderArray.js

```diff
@@ -0,0 +1,23 @@
+const headerTransactionsTitles = [
+    {
+        label: "id",
```

Comment:
aren't these extracted from the json? do we really need to keep them here?

Reply:
created constant which extracts the table head titles based on the transaction list